apps/apple/FluentUITester/README.md

```diff
@@ -0,0 +1,33 @@
+# FluentUI React Native
```

Comment:
Is this needed? Or is this duplicate of what lives in higher-up READMEs? If it isn't, should this live upstream? Nothing here seems specific to FluentTester or Apple platforms?

Reply:
Also "FluentUI" -> "Fluent UI". Fluentui when referring to repo name, "Fluent UI" when referring generically to framework